app/src/main/java/nodomain/freeyourgadget/gadgetbridge/activities/DbManagementActivity.java

```diff
@@ -110,8 +101,33 @@ private String getExternalPath() {
         return getString(R.string.dbmanagementactivvity_cannot_access_export_path);
     }
 
+    private void exportShared() {
+        // BEGIN EXAMPLE
+        File myPath = null;
+        try {
+            myPath = FileUtils.getExternalFilesDir();
+            File myFile = new File(myPath, "Export_preference");
+            shared_file.exportToFile(sharedPrefs,myFile,null);
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
These exceptions should probably be promoted to user notifications (e.g. toasts at least).